docs/typography-styles-page.stories.mdx

```diff
@@ -0,0 +1,8 @@
+import { Meta, Story } from '@storybook/addon-docs';
+import { TextStylesPage } from './components/Typography';
+
+<Meta title="Docs/Typography" component={TextStylesPage} />
```

Comment:
I think in the new Storybook folder structure there is no need to include this story (essenssially a duplicate of the one for `TextStylesPage`, right?